osu.Game/Overlays/Chat/ChannelList/ChannelListSelector.cs

```diff
@@ -50,7 +50,7 @@ private void load(OverlayColourProvider colourProvider)
                     {
                         Anchor = Anchor.CentreLeft,
                         Origin = Anchor.CentreLeft,
-                        Text = "Add More Channels",
+                        Text = "Add more channels",
```

Comment:
Why is this class not a `ChannelListItem`? It's reimplementing much of the same thing. It's also not getting the new selected state change.

Reply:
Having the selector be a separate component was the easiest way to implement it at the time in #17833 but it does seem that it should be refactored to inherit from a base `ChannelListItem`.
Unsure if it's worth including that in this PR or should be updated separately.

Reply:
I mean if it were up to me then this diff would be half the current size, although I do realise that half of the diff is test backporting. I'd try to make the change here and see how large it gets, maybe there's a chance it'll be easy enough to review...

Reply:
I had a go at doing this but I felt it would make a mess of this PR including it here so for now I've just updated the `ChannelListSelector` to turn the text white on select same as the items.